seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestConstant.java

```diff
@@ -39,9 +41,13 @@ public class RestConstant {
 
     public static final String JAR_PATH = "jarPath";
 
+    public static final String ERROR_MSG = "errorMsg";
+
     public static final String METRICS = "metrics";
     public static final String RUNNING_JOBS_URL = "/hazelcast/rest/maps/running-jobs";
     public static final String RUNNING_JOB_URL = "/hazelcast/rest/maps/running-job";
+
+    public static final String FINISHED_JOBS_INFO = "/hazelcast/rest/maps/finished-jobs-info";
```

Comment:
```suggestion
    public static final String FINISHED_JOBS_INFO = "/hazelcast/rest/maps/finished-jobs";
```

Use `finished-jobs`?

Reply:
Done.Thank you for your reminder.